src/commands/games/freefire/weapon.js

```diff
@@ -0,0 +1,61 @@
+const { Command, CommandError, SwitchbladeEmbed } = require('../../../')
+
+const Fuse = require('fuse.js')
+const fetch = require('node-fetch')
+
+module.exports = class FreeFireWeapon extends Command {
+  constructor (client) {
+    super({
+      name: 'weapon',
+      aliases: ['w'],
+      parent: 'freefire',
+      parameters: [{
+        type: 'string', full: true, missingError: 'commands:freefire.subcommands.weapon.missingWeapon'
+      }]
+    }, client)
+  }
+
+  async run ({ t, channel, language }, query) {
+    const { locales } = await this.getData('metadata.json')
+    const locale = locales.find(locale => locale === language.slice(0, 2))
+
+    const { weapons } = await this.getData(`${locale}/weapons.json`)
```

Comment:
We could optimize this by creating a Free Fire API class in order to cache this information and not have to make requests every time ([much like the Legends of Runeterra class I wrote for the card command](https://github.com/SwitchbladeBot/switchblade/blob/dev/src/apis/LegendsOfRuneterra.js))